packages/store-ui/src/index.ts

```diff
@@ -23,6 +23,9 @@ export type { CheckboxProps } from './atoms/Checkbox'
 export { default as Overlay } from './atoms/Overlay'
 export type { OverlayProps } from './atoms/Overlay'
 
+export { default as Radio } from './atoms/Radio'
+export type { RadioProps } from './atoms/Radio'
```

Comment:
Just to remember, don't forget to fix this import path, since I don't know what PR will merge first hahahahah.

Reply:
yes, I'll fix it, I think that only marin brasil uses this component.